### PR TITLE
Fix your first 3D game issues

### PR DESCRIPTION
**Pull request by skyace65**
Fixes some of the issues in #6744

I'm not going to be able to work on this for a few days so I figured it would make sense to mark this for review now, and everything else I'll try and fix in a future PR. In addition to what's mentioned in this issue too many of the new webp images have compression artifacts and should be replaced, I'll see about doing it when I have time

Other changes not from that issue, or fixed in different ways:
- I've updated one of the animation player screenshots for Godot 4
- Changed how font is added in one section. FontVariation doesn't need to be used, a font can just be directly dropped in
- undid "body" being renamed to "_body" in one part of the code, don't know what that was done

### CODE REVIEW

File: getting_started/first_3d_game/07.killing_player.rst [hunk 1]
```diff
@@ -106,7 +106,7 @@ the character should die when an enemy runs into the collider. Note that without
 .. tabs::
    .. code-tab:: gdscript GDScript
 
-    var player_position = $Player.transform.origin
+    var player_position = $Player.position.origin
```
Comment (andreymal) [inline getting_started/first_3d_game/07.killing_player.rst:109]: Just `$Player.position`, without `.origin` https://docs.godotengine.org/en/latest/classes/class_node3d.html#class-node3d-property-position

**Comment by skyace65** [comment]
@andreymal want to take a look at this?

**Review by mhilbrunner — APPROVED** [review]
Changes look good to me :)

**Comment by YuriSizov** [comment]
Upon merging would be work to go over the list of issues and mark with checkboxes which points were addressed here (since not all were). It's very neat that the report is already formatted for that :)

**Comment by skyace65** [comment]
> Upon merging would be good to go over the list of issues and mark with checkboxes which points were addressed here (since not all were). It's very neat that the report is already formatted for that :)

I've been keeping track of which ones I'm fixing so I'll be able to do that.

**Review by Piralein — APPROVED** [review]
only the file size of the images seems a little bit large.

**Comment by Calinou** [comment]
The screenshots were taken with non-default editor theme settings (Border Size is 1, but should be 0).

**Comment by skyace65** [comment]
@Calinou I've updated the animation player and color picture images with new ones with the border size set to 0

**Review by Calinou — APPROVED** [review]

**Comment by clayjohn** [comment]
Looks good and has been reviewed multiple times, so going ahead and merging now!

**Comment by andreymal** [comment]
@clayjohn all my comments have been completely ignored, okay

**Comment by andreymal** [comment]
A, I didn't know github doesn't show my comments until I click "Start review", sorry

**Comment by Piralein** [comment]
No worries, there is always a chance to patch stuff up. ;D

I can do a follow-up PR with some more changes and your review comments, but not sure if I will finish it before weekend.

**Pull request merged**